sites/platform/static/files/fetch/examples/relationships/mysql

```diff
@@ -1,20 +1,24 @@
 {
     "username": "user",
-    "scheme": "mysql",
-    "service": "mariadb104",
     "fragment": null,
     "ip": "169.254.255.221",
-    "hostname": "e3wffyxtwnrxujeyg5u3kvqi6y.mariadb104.service._.eu-3.platformsh.site",
-    "port": 3306,
     "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "mysql.internal",
-    "rel": "mysql",
     "path": "main",
     "query": {
         "is_master": true
     },
     "password": "",
+    "port": 3306,
+    "host_mapped": false,
+    "service": "mariadb104",
+    "hostname": "e3wffyxtwnrxujeyg5u3kvqi6y.mariadb104.service._.eu-3.platformsh.site",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.mariadb104.service._.eu-1.platformsh.site",